processor/src/host/advice/map.rs

```diff
@@ -0,0 +1,53 @@
+use super::{BTreeMap, Felt, Vec};
+use vm_core::utils::collections::btree_map::IntoIter;
+
+use vm_core::crypto::hash::RpoDigest;
+
+//////////////////////////////////
+/// ADVICE MAP
+//////////////////////////////////
+/// This is one of the advice input types
+/// it holds private advice inputs that can be used by the prover
+/// it is a key mapped element list which can be pushed onto the advice stack
+#[derive(Debug, Clone, Default)]
+pub struct AdviceMap(BTreeMap<RpoDigest, Vec<Felt>>);
```

Comment:
Let's change this section to look something like this (to have consistent format across the repo):
```Rust
// ADVICE MAP
// ================================================================================================

/// Defines a set of non-deterministic (advice) inputs which the VM can access by their keys.
///
/// Each key maps to one or more field element. To access the elements, the VM can move the values
/// associated with a given key onto the advice stack using `adv.push_mapval` instruction. The VM
/// can also insert new values into the advice map during execution. 
#[derive(Debug, Clone, Default)]
pub struct AdviceMap(BTreeMap<RpoDigest, Vec<Felt>>);
```